cg/meta/workflow/mutant/metrics_parser/models.py

```diff
@@ -0,0 +1,33 @@
+from typing import Annotated
+from pydantic import BaseModel, BeforeValidator, ValidationError
+
+
+def empty_str_to_none(v: str) -> str | None:
+    return v or None
+
+
+def str_to_bool(v: str) -> bool:
+    if v == "TRUE":
+        return True
+    elif v == "FALSE":
+        return False
+    else:
+        raise ValidationError
```

Comment:
```suggestion
def str_to_bool(value: str) -> bool:
    if value.upper() == "TRUE":
        return True
    elif value.upper() == "FALSE":
        return False
    raise ValidationError
```
You don't need the else, the function returns if either of the above is true. or it should raise the error.